ext/opcache/jit/zend_jit.c

```diff
@@ -39,7 +39,14 @@
 #include "Optimizer/zend_call_graph.h"
 #include "Optimizer/zend_dump.h"
 
+#if defined(__x86_64__) || defined(i386) || defined(ZEND_WIN32)
```

Comment:
@nikic I didn't get what do you mean. Change conditions order? Check ``#ifdef defined (__aarch64__)`` before ``#if defined(__x86_64__) || defined(i386) || defined(ZEND_WIN32)``?

Reply:
I was thinking something along the lines of `#define ZEND_JIT_IS_X86 defined(__x86_64__) || defined(i386) || defined(ZEND_WIN32)` and use that as the condition rather than repeating the three checks. This is just a style suggestion, it's not important.

(I doubt JIT on AArch64 Windows would work out of the box, so current check seems fine for now.)

Reply:
I introduced ZEND_JIT_TARGET_X86 and ZEND_JIT_TARGET_ARM64.